fuzzers/030-iob18/README.md

```diff
@@ -0,0 +1,2 @@
+# IOB18 Fuzzer
```

Comment:
Would be good to have something here

Reply:
OK rebased to current master and review notes applied.